pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -24,8 +23,22 @@ var mercuryInfo = capabilities.MustNewCapabilityInfo(
 
 const defaultTickerResolutionMs = 1000
 
+type mercuryTriggerConfig struct {
+	// strings should be hex-encoded 32-byte values, prefixed with "0x", all lowercase, minimum 1 item
+	FeedIDs []string `json:"feedIds" jsonschema:"pattern=^0x[0-9a-f]{64}$,minItems=1"`
+	// must be greater than 0
+	MaxFrequencyMs int `json:"maxFrequencyMs" jsonschema:"minimum=1"`
+}
+
+type mercuryTriggerInputs struct {
```

Comment:
Nit: you can avoid some stuttering here since these are private:
mercuryTriggerConfig/Inputs -> config/inputs

Reply:
+1